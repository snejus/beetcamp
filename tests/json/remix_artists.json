{
  "@context": "https://schema.org",
  "@id": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 1571280308
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 1
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 832154543
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 1994130301
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 1571280308
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a1571280308_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "UNREALNUMBERS - Unseen EP (Varya Karpova & Lacchesi Remixes)"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes#b58770622",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 58770622
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "b"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 1994130301
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 1045504419
        },
        {
          "@type": "PropertyValue",
          "name": "is_bfd",
          "value": true
        }
      ],
      "image": [
        "https://f4.bcbits.com/img/a1045504419_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "full digital discography (21 releases)",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "bundle_size",
            "value": 21
          },
          {
            "@type": "PropertyValue",
            "name": "discount",
            "value": 0.4
          }
        ],
        "availability": "OnlineOnly",
        "price": 106.2,
        "priceCurrency": "EUR",
        "priceSpecification": {
          "minPrice": 106.2
        },
        "url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes#b58770622-buy"
      }
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-karaburan",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-rekon",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen-varya-karpova-remix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4-lacchesi-remix",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@id": "https://maisoncloserecords.bandcamp.com",
    "@type": "MusicGroup",
    "name": "Maison Close"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "11 Jun 2021 08:55:23 GMT",
  "datePublished": "11 Jun 2021 00:00:00 GMT",
  "description": "Supported by Dax J, SPFDJ, Wallis, u.r.trax, Rezystor, DYEN, VII Circle, VII Circle & more ...",
  "image": "https://f4.bcbits.com/img/a1571280308_10.jpg",
  "keywords": [
    "Electronic",
    "techno",
    "Paris"
  ],
  "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
  "name": "UNREALNUMBERS - Unseen EP (Varya Karpova & Lacchesi Remixes)",
  "numTracks": 6,
  "publisher": {
    "@id": "https://maisoncloserecords.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 1994130301
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 2000
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 31964905
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 2000
      }
    ],
    "foundingLocation": {
      "@type": "Place",
      "name": "Paris, France"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0031964905_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebPage",
        "name": "Facebook",
        "url": "https://www.facebook.com/maisoncloserecords"
      }
    ],
    "name": "Maison Close",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://maisoncloserecords.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "merch",
        "url": "https://maisoncloserecords.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://maisoncloserecords.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3030790431
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M23S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen",
          "name": "UNREALNUMBERS - Unseen"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 987420966
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M52S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4",
          "name": "UNREALNUMBERS - MK4"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-karaburan",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3850977436
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M41S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-karaburan",
          "name": "UNREALNUMBERS -Karaburan"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-rekon",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 628289803
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M40S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-rekon",
          "name": "UNREALNUMBERS - Rekon"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen-varya-karpova-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3533046617
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Varya Karpova"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M02S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen-varya-karpova-remix",
          "name": "UNREALNUMBERS - Unseen (Varya Karpova Remix)"
        },
        "position": 5
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4-lacchesi-remix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 359910911
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Lacchesi"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M50S",
          "mainEntityOfPage": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4-lacchesi-remix",
          "name": "UNREALNUMBERS - MK4 (Lacchesi Remix)"
        },
        "position": 6
      }
    ],
    "numberOfItems": 6
  }
}
