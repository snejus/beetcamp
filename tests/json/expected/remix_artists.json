[
  {
    "album": "Unseen EP",
    "album_id": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "ep",
    "albumtypes": [
      "ep",
      "remix"
    ],
    "artist": "UNREALNUMBERS",
    "artist_credit": null,
    "artist_id": "https://maisoncloserecords.bandcamp.com",
    "artist_sort": null,
    "artists": [
      "UNREALNUMBERS"
    ],
    "artists_credit": [],
    "artists_ids": [
      "https://maisoncloserecords.bandcamp.com"
    ],
    "artists_sort": [],
    "asin": null,
    "barcode": null,
    "catalognum": "",
    "comments": "Supported by Dax J, SPFDJ, Wallis, u.r.trax, Rezystor, DYEN, VII Circle, VII Circle & more ...",
    "country": "FR",
    "data_source": "bandcamp",
    "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
    "day": 11,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": "techno",
    "label": "Maison Close",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 6,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "release_group_title": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 383,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Unseen",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 352,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 6,
        "release_track_id": null,
        "title": "MK4",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 341,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Karaburan",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-karaburan",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 340,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Rekon",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-rekon",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 362,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 6,
        "release_track_id": null,
        "title": "Unseen (Varya Karpova Remix)",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-unseen-varya-karpova-remix",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Unseen EP",
        "arranger": null,
        "artist": "UNREALNUMBERS",
        "artist_credit": null,
        "artist_id": "https://maisoncloserecords.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "UNREALNUMBERS"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://maisoncloserecords.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://maisoncloserecords.bandcamp.com/album/unrealnumbers-unseen-ep-varya-karpova-lacchesi-remixes",
        "disctitle": null,
        "genre": null,
        "index": 6,
        "initial_key": null,
        "length": 350,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 6,
        "medium_total": 6,
        "release_track_id": null,
        "title": "MK4 (Lacchesi Remix)",
        "track_alt": null,
        "track_id": "https://maisoncloserecords.bandcamp.com/track/unrealnumbers-mk4-lacchesi-remix",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2021
  }
]
